requirements.txt

```diff
@@ -0,0 +1 @@
+semver==2.8.1
```

Comment:
What about `xml`? Also this should be stored in the `Tools/jenkins` directory (from https://github.com/Leanplum/Leanplum-Android-SDK/pull/232/files) when it lands

Reply:
It's a part of the std lib so nothing to be installed.